### PR TITLE
add pwas to the managebility policy of sidebar

### PR DESCRIPTION
**Pull request by jahn00**
Update docs to include PWAs in the `HubsSidebarEnabled` manageability policy of Edge.

### CODE REVIEW

File: edgeenterprise/microsoft-edge-sidebar.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Admins have several policy options for deploying and managing the sidebar in the
 
 ## Allow or block the sidebar in group policy
 
-You can use [HubsSidebarEnabled](/deployedge/microsoft-edge-policies#hubssidebarenabled)  policy to control whether the sidebar is allowed or blocked in your organization. Blocking the sidebar will automatically block all sidebar apps from being enabled.
+You can use [HubsSidebarEnabled](/deployedge/microsoft-edge-policies#hubssidebarenabled)  policy to control whether the sidebar is allowed or blocked in your organization. Blocking the sidebar will automatically block all sidebar apps from being enabled. This policy also simultaneously allows or blocks the sidebar within PWAs or [Progressive Web Apps](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/).
```
Comment (HowardWolosky) [inline edgeenterprise/microsoft-edge-sidebar.md:39]: You may need to clarify which Edge version this policy _also_ impacts the PWA sidebar.

Comment (jahn00) [reply edgeenterprise/microsoft-edge-sidebar.md]: The PR to introduce the policy isn't in main yet, so I cannot provide an exact version. Is Edge 116 or after sufficient?

File: edgeenterprise/microsoft-edge-sidebar.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Admins have several policy options for deploying and managing the sidebar in the
 
 ## Allow or block the sidebar in group policy
 
-You can use [HubsSidebarEnabled](/deployedge/microsoft-edge-policies#hubssidebarenabled)  policy to control whether the sidebar is allowed or blocked in your organization. Blocking the sidebar will automatically block all sidebar apps from being enabled.
+You can use [HubsSidebarEnabled](/deployedge/microsoft-edge-policies#hubssidebarenabled)  policy to control whether the sidebar is allowed or blocked in your organization. Blocking the sidebar will automatically block all sidebar apps from being enabled. For versions of Edge 116 and onward, this policy also simultaneously allows or blocks the sidebar within PWAs or [Progressive Web Apps](https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/).
```
Comment (HowardWolosky) [inline edgeenterprise/microsoft-edge-sidebar.md:39]: Looking at the other policies, it seems that the agreed upon way of phrasing that would be "On Edge 116 or later, this policy..."

Comment (Y-Kelly) [inline edgeenterprise/microsoft-edge-sidebar.md:39]: Suggested wording update from @dan-wesley (content manager)

... allows or blocks the sidebar in [Progressive Web Apps (PWAs)](/microsoft-edge/progressive-web-apps-chromium).

Comment (jahn00) [reply edgeenterprise/microsoft-edge-sidebar.md]: Fixed wording and added relative link.

**Comment by Y-Kelly** [comment]
@jahn00 We can publish this update once Beta 116 gets released.

**Review by ngaitonde — APPROVED** [review]

**Pull request merged**